.github/workflows/test.yml

```diff
@@ -1,31 +1,130 @@
-# This workflow will install Python dependencies, run tests and lint with a single version of Python
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
-
 name: Run Tests
 
+# Explicitly set permissions to allow Dependabot workflow runs to write in the PR
+# for coverage's reporting.
+# By default, these are read-only when the actions are ran by Dependabot
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#changing-github_token-permissions
+permissions: write-all
+
+env:
+  ALL_PYTHON_VERSIONS: "['3.10', '3.11']"
```

Comment:
I don't think we need to lint on all Python versions. I think we should run the tests on all supported Python versions and lint on the minimum supported version.